app/controllers/OrganizationController.js

```diff
@@ -0,0 +1,40 @@
+var blueprint = require ('@onehilltech/blueprint')
+  , mongodb = require ('@onehilltech/blueprint-mongodb')
+  , ResourceController = mongodb.ResourceController
+  , messaging = blueprint.messaging
+  ;
+
+var Organization = require ('../models/Organization')
+  ;
+
+// this code effectively subclasses ResourceController with the specific information below
+function OrganizationController () {
+  ResourceController.call (this, {name: 'organization', model: Organization});
+}
+
+blueprint.controller (OrganizationController, ResourceController);
+
+// preCreate function to verify if organization has already been created
+OrganizationController.prototype.create = function () {
+  var opts = {
+    on: {
+      preCreate: function (req, doc, callback) {
+        Organization.findOne({ name: req.body.name }, function (err, organization) {
+          if (err) { return callback (err); }
+
+          if (organization){
+            return callback (new Error ('Organization already exists'), doc);
+          } else {
+            return callback (null, doc);
+          }
+        });
+      },
+      postExecute: function () {
+        messaging.emit('organization.created', organization)
```

Comment:
The organization variable has not not been declared in the scope of this function.  After looking at some of Dr. Hill's examples I believe the postExecute function is declared as follows:
```
postExecute: function (req, result, callback) {
        return callback (null, result);
}
```
I would experiment with this and try printing the result object to see what that value is going to be.